setup.sh

```diff
@@ -10,34 +10,32 @@ debian_packages=(qemu git tar python3-docutils pulseaudio-utils dunst sphinx-doc
 SILENT=1
 VERBOSE=0
 FORCE=0
-PREFIX="${PREFIX:-$HOME/.local}"
 
 declare -r app_name="kw"
 
 ##
 ## Following are the install paths
 ##
 # Paths used during the installation process
-declare -r kwbinpath="$PREFIX/bin/kw"
-declare -r binpath="$PREFIX/bin"
-declare -r libdir="$PREFIX/lib/$app_name"
-declare -r sharedir="$PREFIX/share/"
+declare -r kwbinpath="$HOME/.local/bin/$app_name"
+declare -r binpath="$HOME/.local/bin"
+declare -r libdir="$HOME/.local/lib/$app_name"
+declare -r sharedir="${XDG_DATA_HOME:-"$HOME/.local/share"}/$app_name"
```

Comment:
Why did you remove `PREFIX`? I think it looks better with that...

Reply:
What I originally thougth is that "$PREFIX" conveys the idea that all of kw is installed under one directory, which is not true anymore. Also, we start working with a lot of variables at the same time (see example below). For me is a little bit less clear what's going on. What do you think?

```
PREFIX="$HOME/.local"
KW_SHARE="${XDG_DATA_HOME:-"$PREFIX/share"}/$KWORKFLOW"
KW_SHARE_DOC_DIR="$KW_SHARE/$KWORKFLOW"
```

Reply:
Ok, I agree.